great_expectations/self_check/util.py

```diff
@@ -840,25 +842,30 @@ def _get_test_validator_with_data_spark(  # noqa: C901 - 19
                 print(schema)
             for col in schema:
                 type_ = schema[col]
+                # Ints cannot be None...but None can be valid in Spark (as Null)
+                vals: List[Union[str, int, float, None, Decimal]] = []
```

Comment:
@TrangPham If we are going to have a type hint for `vals` here, should we also have it for all cases of `type_` below?  Thanks.

Reply:
This would apply to all of them. I cannot redefine the variable with its type again, doing so causes an error with the static type checking in CI pipeline.